README.md

```diff
@@ -59,3 +59,9 @@ Express the radius as a solution of a differential equation:
           <*> derivative' seconds x  
           <*> click 
 ```
+The differential equation is _d<sup>2</sup>x/dt<sup>2</sup> = -5(x - 50) - 2dx/dt_
+,  which is rearranged so the right hand side gives an expression whose fixed point solution is a function of time describing an oscillation.
+
+y = ∫ ∫  _-5(x - 50) - 2dx/dt_ dt dt
```

Comment:
You have `x` on the right but `y` on the left here.

Reply:
@paf31  sorry. typo corrected, by rewriting commit